adops/adops-planning-guide.md

```diff
@@ -63,6 +63,28 @@ Some additional resources that might be helpful as you work through your setup i
 
 [Deals in Prebid](/adops/deals.html): You can negotiate deals with header bidding demand partners and have them compete with ad server inventory. You’ll need to create additional line items to support these deals. This document explains how to set up Prebid to make that happen.
 
+## Special Considerations
+
+### Rendering mobile outstream ads
+
+The 'outstream' format requires different ways of rendering for Prebid.js vs Mobile:
+
+- For Prebid.js web integrations, outstream ads are rendered like banners.
```

Comment:
This isn't accurate. To render outstream you can use the same tech used for instream. It seems that many publishers using Prebid.js don't have an outstream player and rely on HTML being delivered with a renderer, but this is not a requirement and we shouldn't state it as so. If a Publisher has an outstream player, you don't want to get HTML you just want a VAST tag or a VAST xml.

Reply:
> To render outstream you can use the same tech used for instream.

I disagree. For instream and outsteam-in-mobile, the GAM setup requires a video line item with a VastUrl creative. For outstream-in-prebid.js, the GAM setup requires a display line item and the PUC or alternate.

So I'm willing to change "rendered like banners." to "set up in the ad server like banners." would that help?

Reply:
I don't understand why prebid.js needs outstream video to be setup as a display line item. Perhaps it would be easier to discuss tomorrow at the Video PMC meeting. 
Technology wise, rendering an outstream video ad is the same as rendering an instream video ad, given a video player that supports both instream and outstream video ads. So i'm confused why GAM would need it to be setup as a display line item.

Reply:
I admit I haven't spent much time with GAM and line items. Does GAM not treat outstream as video by definition?